src/data-structure.c

```diff
@@ -106,6 +106,12 @@ uint32_t* bitmap_get_int_array(Bitmap* bitmap, size_t* n) {
   roaring_bitmap_to_uint32_array(bitmap, ans);
   return ans;
 }
+uint32_t* bitmap_range_int_array(Bitmap* bitmap, size_t offset,  size_t n) {
+  uint32_t* ans = malloc(sizeof(*ans) * n);
+  memset(ans, 0, sizeof(*ans) * n);
```

Comment:
Maybe these two lines could be a `calloc`?

```C
uint32_t* ans = calloc(n, sizeof(*ans));
```

Reply:
very thanks